### DISCUSSION TITLE
QTable Quasar Sticky header and column scroll not working on IPad

### DISCUSSION THREAD

**Discussion by pratik227** [quasarframework/quasar#13006] [General - Components / Directives / etc]
Codepen - https://codepen.io/Pratik__007/pen/PoEJYbR

I tried to add a sticky header and sticky column but on IPad, it doesn't work properly.

https://user-images.githubusercontent.com/34883558/161042775-b67be1a8-7b58-43c0-9acd-46e9766619b2.MOV

Any Solution for this?

**Comment by pdanpdan** [comment]
It looks like you need to test applying some z-index on the thead/tr/th - see what ipad wants

**Reply by pratikpatel-aurochs** [reply]
@pdanpdan The issue is with scroll. I have tried to change the z-index but didn't work. You can see in the video if I scroll down it also scrolls right side. I have tried the default quasar codepen on Ipad and but it's also not working.

**Comment by pdanpdan** [comment]
The scroll is completely native

**Reply by pratikpatel-aurochs** [reply]
Sorry, but I didn't get it. Is there any way to fix it?

**Comment by suleimanas** [comment]
if you mean the first column, set position sticky to important
```

.my-sticky-column-table th:first-child, td:first-child {
  position: sticky !important;
  ...
}
```

that'll fix it

**Reply by pratikpatel-aurochs** [reply]
No, it's not working and in the table I have 4 sticky columns. You can see the issue is with the scroll. When you scroll left it also scrolls the bottom side. I don't know how to fix it. pdanpdan said it's native but I don't know why it's happening.

**Reply by pratikpatel-aurochs** [reply]
https://codepen.io/Pratik__007/pen/PoEJYbR

**Reply by suleimanas** [reply]
sorry still don't get you :), the video shows normal scrolling behavior
